### DISCUSSION TITLE
Duotone glyphs

### DISCUSSION THREAD

**Discussion by haithium** [ryanoasis/nerd-fonts#1592] [Ideas]
I was wondering if FontForge supports opacity on glyphs so we can use icons like [Font Awsome duotone](https://fontawesome.com/search?o=r&s=duotone) in the glyphs.

![image](https://github.com/ryanoasis/nerd-fonts/assets/128622475/01bc84b4-b2e6-4af4-9194-9d37815ab7c0)

**Comment by Finii** [comment]
I believe this is not opacity or something, but two ordinary glyphs you print on top of each other with different colors.

For eaxmple the house icon, there is one glyph with just the roof and one glyph just the wall, and then you can combine these two by printing on top of each other with any color combo you like.
